restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/client/api/ConferenceApiClient.java

```diff
@@ -0,0 +1,173 @@
+ *
+ */
+public class ConferenceApiClient extends RestcommUntypedActor {
+
+    private final LoggingAdapter logger = Logging.getLogger(getContext().system(), this);
+
+    private final DaoManager storage;
+    private final String accountSid;
+    private final String name;
+    private final Sid conferenceSid;
+    private ConferenceDetailRecord conferenceDetailRecord;
+    private ActorRef requestee;
+
+    private ActorRef httpAsycClientHelper;
+
+    public ConferenceApiClient(final String accountSid, final String name, final Sid conferenceSid, final DaoManager storage) {
+        super();
+        this.accountSid = accountSid;
+        this.name = name;
+        this.conferenceSid = conferenceSid;
+        this.storage = storage;
+        //actor will only live in memory for one hour
+        context().setReceiveTimeout(Duration.create(3600, TimeUnit.SECONDS));
+    }
+    @Override
+    public void onReceive(Object message) throws Exception {
+        final Class<?> klass = message.getClass();
+        final ActorRef sender = sender();
+        ActorRef self = self();
+
+        if (logger.isInfoEnabled()) {
+            logger.info(" ********** ConferenceApiHelper " + self().path() + " Sender: " + sender);
+            logger.info(" ********** ConferenceApiHelper " + self().path() + " Processing Message: " + klass.getName());
+        }
+        if (StopConference.class.equals(klass)) {
+            onStopConference((StopConference) message, self, sender);
+        } else if (DownloaderResponse.class.equals(klass)) {
+            onDownloaderResponse(message, self, sender);
+        } else if (message instanceof ReceiveTimeout) {
+            getContext().stop(self());
+        }
+    }
+
+    private void onDownloaderResponse(Object message, ActorRef self, ActorRef sender) {
+        final DownloaderResponse response = (DownloaderResponse) message;
+        if (logger.isInfoEnabled()) {
+            logger.info("Conference api response succeeded " + response.succeeded());
+            logger.info("Conference api response: " + response);
+        }
+        requestee.tell(message, self);
+        // since this is a one time disposable client lets clean it off after it sends DownloaderResponse.
+        getContext().stop(self());
+    }
+    protected void onStopConference(StopConference message, ActorRef self, ActorRef sender) throws URISyntaxException, ParseException {
+        requestee = sender;
+        conferenceDetailRecord = getConferenceDetailRecord();
+        if(conferenceDetailRecord == null){
+            logger.error("could not retrieve cdr by provided Sid");
+            onDownloaderResponse(new DownloaderResponse(new IllegalArgumentException("could not retrieve cdr by provided Sid")), self, sender);
+        } else {
+            try {
+                URI uri = new URI("/restcomm"+conferenceDetailRecord.getUri());
+                uri = UriUtils.resolve(uri);
```

Comment:
@maria-farooq here we rely on the fact that the cluster has a common public IP address (the LB public IP). Nothing bad in the current phase but maybe the `conferenceDetailRecord.getUri()` should contain the remote IP Address (when/if the conference is remote).
This would help later to have conferences between instances that don't belong to the same "cluster". 
This is just an idea for discussion.